test/collection_test.js

```diff
@@ -1496,25 +1512,22 @@ describe("Collection", () => {
         ]);
     });
 
-    it("should batch deletion of locally deleted records", (done) => {
-      const batch = sandbox.stub(articles.api, "batch").returns(Promise.resolve({
-        published: [],
-        errors:    [],
-        conflicts: [],
-        skipped:   [],
-      }));
+    it("should batch deletions of locally deleted records", () => {
```

Comment:
Rephrase this test, we're actually testing that never synced locally deleted records are not published to the server.